Source/SoundEffect.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SoundEffect.swift
+//  ImagePicker
+//
+//  Created by Zachary Steed on 9/1/17.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
```

Comment:
Maybe remove this boilerplate?

Reply:
Yeah I would remove this

Reply:
But I can't see this in the overall diff. Maybe GitHub is playing tricks on me.